sites/cheerpx/src/content/docs/00-overview.mdx

```diff
@@ -5,21 +5,61 @@ description: Native code virtualization as a JavaScript library
 ---
 
 import LinkButton from "@leaningtech/astro-theme/components/LinkButton.astro";
+import { DISCORD_URL } from "@/consts.ts";
+
+<div class="not-prose flex gap-2 mb-2">
+	<img src="https://img.shields.io/badge/version-1.0-blue" alt="Version 1.0" />
+	<a href="https://discord.leaningtech.com/">
+		<img
+			src="https://img.shields.io/discord/988743885121548329?color=%237289DA&logo=discord&logoColor=ffffff"
+			alt="Discord server"
+		/>
+	</a>
+	<a href="https://github.com/leaningtech/labs">
+		<img
+			src="https://img.shields.io/badge/repository-labs-yellow?logo=github"
+			alt="GitHub repository"
+		/>
+	</a>
+	<a href="https://github.com/leaningtech/labs/issues">
+		<img
+			src="https://img.shields.io/github/issues/leaningtech/labs?logo=github"
+			alt="GitHub issues"
+		/>
+	</a>
+</div>
 
 **CheerpX is an x86 virtualization technology for running executables and operating systems entirely client-side.** It's an always available, zero-cost virtual machine with guaranteed data privacy, powered by a WebAssembly JIT engine.
 
-<div class="flex items-center gap-2">
-  <LinkButton
-    label="Get started"
-    iconRight="jam:chevron-right"
-    type="primary"
-    href="/docs/getting-started"
-  />
+<div class="flex items-center gap-2 justify-center">
+	<LinkButton
+		label="WebVM Demo"
+		iconLeft="mi:play"
+		type="secondary"
+		href="https://webvm.io"
+	/>
+</div>
+
+<div class="m-4 flex justify-center">
+	[Get started](/docs/getting-started) · [Guides](/docs/guides) · [API
+	Reference](/docs/reference) · [Tutorials](/docs/tutorials) · [Github
+	repository](https://github.com/leaningtech/labs)
+</div>
+
+```html
+<script src="https://cxrtnc.leaningtech.com/1.0.0/cx.js"></script>
+```
 
-  <LinkButton label="Demo" iconLeft="mi:play" type="secondary" href="https://webvm.io" />
+<div class="text-stone-400">
+	Not at the right address? Visit our [website](https://leaningtech.com/) to
```

Comment:
I am confused by this message. It suggests to me that the loader URL is not the right address. This message (rephrased, something like "Not what you were looking for?" should go above after the very first introduction paragraph